### PR TITLE
Correct the nginx X-Forwarded-Proto lines so that rails request.protocol...

### PR DESCRIPTION
**Pull request by kirtfitzpatrick**

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: templates/default/supermarket.nginx.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@ server {
   proxy_set_header        Host            $host;
   proxy_set_header        X-Real-IP       $remote_addr;
   proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-  proxy_set_header        X-Forwarded-Proto https;
+  proxy_set_header        X-Forwarded-Proto http;
```
Comment (raskchanky) [inline templates/default/supermarket.nginx.erb]: I might be missing some context here, but if `node['supermarket']['force_ssl']` is `true`, why would we want to set the protocol to `http`?

**Comment by kirtfitzpatrick** [comment]
This was in reference to issue #994

**Comment by raskchanky** [comment]
So yes, I was missing some context, and now that I have the correct context - :+1:

**Comment by kisoku** [comment]
@kirtfitzpatrick  Any chance you guys would consider cutting a new release of the cookbook with this fix.

**Comment by kirtfitzpatrick** [comment]
@kisoku done.

**Pull request merged**